app/assets/stylesheets/notice.scss

```diff
@@ -0,0 +1,17 @@
+.notice {
```

Comment:
Similarly to `forms.scss` case: we have lots of `.notice` elements elsewhere - make sure they all work correctly.